doc/memos/rdm0.md

```diff
@@ -0,0 +1,126 @@
+community processes, position with respect to some related external technical 
+debates etc.
+
+
+This memo specifies a format for such memos and the process of discussing their 
+content before publication. Format and processes are somewhat inspired by 
+[IETF](http://ietf.org/) processes and documents such as Request For Comments 
+(RFC).
+
+# 2. RIOT Developers Memo Format
+
+RIOT Developer Memos (RDM) are written in English, using the Markdown (.md) 
+format. Each memo has a unique identification number indicated in its header, 
+e.g. this memo is RDM0.
+
+The structure of a memo should contain:
+
+1. a preamble as defined below,
+2. the content of the memo structured as needed,
+3. a postamble as defined below.
+
+The preamble of the memo should be structured as such:
+
+1. a header formated as the header of this memo,
+2. a short abstract,
+3. a status for the memo stating its license and the category of the memo (e.g. 
+that it is a consensus document),
+4. optionally, a terminology section which recalls/disambiguates definition of 
+terms that are key to the memo.
+
+The core content of the memo is structured as deemed necessary. However, during 
+the review process, care must be taken to follow the below principles as much as
+possible:
+
+1. one memo = one problem/aspect,
+2. start by crisply explaining and motivating the one problem/aspect the memo 
+targets,
+3. memos are expected to be of normative and durable nature, hence: keep a memo 
+as concise as possible without impairing clarity, and factor out content that 
+would be likely to be updated too often.
+
+The postamble of the memo should be structured as such:
+
+1. (optional) acknowledgements section thanking contributors other than the 
+authors,
+2. a section with the main references that relate to the content, with a strong
+preference for permanent, self-contained references.
+
+
+# 3. Memo Publishing and Maintenance Process
+
+Each memo is proposed as a pull-request (PR) and discussed the same way RIOT
+code PRs are processed on GitHub.
+
+
+Once proposed, a memo must have received an ACK by at least 3 maintainers other 
+than the author(s) of the memo, before its publication. 
+
+
+Once merged into the master branch of RIOT, a memo is considered published.
+
+Once merged, small changes/clarifications may be subsequently PRed to improve a
+memo. Bigger changes should result in a new memo deprecating the old memo.
+
+A dedicated directory in RIOT codebase (RIOT/doc/memos) contains all the memos.
+It is acceptable to add graphical content (in RIOT/doc/memos/graphics/) to
+illustrate the memo, as done with the RIOT logo below (see Figure 1, given here
+for pure sake of example). However graphical content should be kept to the bare 
+minimum, as it is easy to get lost in "shiny slideware".
+
+
+![Figure 1. RIOT logo.](graphics/rdm0/RIOT-logo.png "RIOT logo")
+<p align="justify">
+_Figure 1. The RIOT logo._
```

Comment:
If I am not mistaken, single asterisk (`*`) around a word will yield an italic font -> *foo*